deploy.sh

```diff
@@ -0,0 +1,170 @@
+			# The .gitattributes file has to be committed to be used
+			# Just don't push it to the origin repo :)
+			git add .gitattributes && git commit -m "Add .gitattributes file"
+		fi
+
+		# This will exclude everything in the .gitattributes file with the export-ignore flag
+		git archive HEAD | tar x --directory="$TMP_DIR"
+
+		cd "$SVN_DIR"
+
+		# Copy from clean copy to /trunk, excluding dotorg assets
+		# The --delete flag will delete anything in destination that no longer exists in source
+		rsync -rc "$TMP_DIR/" trunk/ --delete --delete-excluded
+	fi
+else
+	echo "ℹ︎ Copying files from build directory..."
+	rsync -rc "$BUILD_DIR/" trunk/ --delete --delete-excluded
+fi
+
+# Copy dotorg assets to /assets
+if [[ -d "$GITHUB_WORKSPACE/$ASSETS_DIR/" ]]; then
+	rsync -rc "$GITHUB_WORKSPACE/$ASSETS_DIR/" assets/ --delete
+else
+	echo "ℹ︎ No assets directory found; skipping asset copy"
+fi
+
+# Add everything and commit to SVN
+# The force flag ensures we recurse into subdirectories even if they are already added
+# Suppress stdout in favor of svn status later for readability
+echo "➤ Preparing files..."
+svn add . --force > /dev/null
+
+# SVN delete all deleted files
+# Also suppress stdout here
+svn status | grep '^\!' | sed 's/! *//' | xargs -I% svn rm %@ > /dev/null
+
+# Copy tag locally to make this a single commit
+echo "➤ Copying tag..."
+svn cp "trunk" "tags/$VERSION"
+
+# Fix screenshots getting force downloaded when clicking them
+# https://developer.wordpress.org/plugins/wordpress-org/plugin-assets/
+if test -d "$SVN_DIR/assets" && test -n "$(find "$SVN_DIR/assets" -maxdepth 1 -name "*.png" -print -quit)"; then
+    svn propset svn:mime-type "image/png" "$SVN_DIR/assets/"*.png || true
+fi
+if test -d "$SVN_DIR/assets" && test -n "$(find "$SVN_DIR/assets" -maxdepth 1 -name "*.jpg" -print -quit)"; then
+    svn propset svn:mime-type "image/jpeg" "$SVN_DIR/assets/"*.jpg || true
+fi
+if test -d "$SVN_DIR/assets" && test -n "$(find "$SVN_DIR/assets" -maxdepth 1 -name "*.gif" -print -quit)"; then
+    svn propset svn:mime-type "image/gif" "$SVN_DIR/assets/"*.gif || true
+fi
+if test -d "$SVN_DIR/assets" && test -n "$(find "$SVN_DIR/assets" -maxdepth 1 -name "*.svg" -print -quit)"; then
+    svn propset svn:mime-type "image/svg+xml" "$SVN_DIR/assets/"*.svg || true
+fi
+
+#Resolves => SVN commit failed: Directory out of date
+svn update
+
+svn status
+
+#echo "➤ Committing files..."
+#svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+
+#if $INPUT_GENERATE_ZIP; then
+#  echo "Generating zip file..."
+
+  # use a symbolic link so the directory in the zip matches the slug
+#  ln -s "${SVN_DIR}/trunk" "${SVN_DIR}/${SLUG}"
+#  zip -r "${GITHUB_WORKSPACE}/${SLUG}.zip" "$SLUG"
+#  unlink "${SVN_DIR}/${SLUG}"
+
+#  echo "zip-path=${GITHUB_WORKSPACE}/${SLUG}.zip" >> "${GITHUB_OUTPUT}"
+#  echo "✓ Zip file generated!"
+#fi
```

Comment:
Should this be commented out? As far as I understand this code (not too familiar with GH Actions tbh), it creates a ZIP file for the built plugin (as an artifact)? If that's the case, shouldn't we have this part of the code run already so that we can test that the ZIP file works as expected?

Reply:
This part of the code creates a ZIP file from the release which can then be passed to additional workflow steps. See: https://github.com/10up/action-wordpress-plugin-deploy#inputs. By default, this will be set as `false`, so it it's fine to leave this comment it out. The reason that this probably needed to be commented out, is that our workflow does not set the environment variable, which might make this line result in an error.

Reply:
Never mind, just saw the updated commit that uncomments that code, and it seems fine.

Reply:
Thank you both!